examples/quickstart/jupyter-notebooks/kafka-tutorial.ipynb

```diff
@@ -0,0 +1,753 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\n",
+      "    \"type\": \"kafka\",\n",
+      "    \"spec\": {\n",
+      "        \"ioConfig\": {\n",
+      "            \"type\": \"kafka\",\n",
+      "            \"consumerProperties\": {\n",
+      "                \"bootstrap.servers\": \"kafka:9092\"\n",
+      "            },\n",
+      "            \"topic\": \"social_media\",\n",
+      "            \"inputFormat\": {\n",
+      "                \"type\": \"json\"\n",
+      "            },\n",
+      "            \"useEarliestOffset\": true\n",
+      "        },\n",
+      "        \"tuningConfig\": {\n",
+      "            \"type\": \"kafka\"\n",
+      "        },\n",
+      "        \"dataSchema\": {\n",
+      "            \"dataSource\": \"social_media\",\n",
+      "            \"timestampSpec\": {\n",
+      "                \"column\": \"__time\",\n",
+      "                \"format\": \"iso\"\n",
+      "            },\n",
+      "            \"dimensionsSpec\": {\n",
+      "                \"dimensions\": [\n",
+      "                    \"username\",\n",
+      "                    \"post_title\",\n",
+      "                    {\n",
+      "                        \"type\": \"long\",\n",
+      "                        \"name\": \"views\"\n",
+      "                    },\n",
+      "                    {\n",
+      "                        \"type\": \"long\",\n",
+      "                        \"name\": \"upvotes\"\n",
+      "                    },\n",
+      "                    {\n",
+      "                        \"type\": \"long\",\n",
+      "                        \"name\": \"comments\"\n",
+      "                    },\n",
+      "                    \"edited\"\n",
+      "                ]\n",
+      "            },\n",
+      "            \"granularitySpec\": {\n",
+      "                \"queryGranularity\": \"none\",\n",
+      "                \"rollup\": false,\n",
+      "                \"segmentGranularity\": \"hour\"\n",
+      "            }\n",
+      "        }\n",
+      "    }\n",
+      "}\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(json.dumps(json.loads(kafka_ingestion_spec), indent=4))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<Response [200]>"
```

Comment:
might be nicer if we had a pretty output here instead of this response looking object.

Reply:
Not sure how to make this nicer, but I can add some text after it with some details.